mvs/ACMMP/adapter.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022, Arabella Gromova, Ivan Moskalenko, Kirill Ivanov, Anastasiia Kornilova
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from common.adapter_base import AdapterBase
+import subprocess
+import os
+
+class Adapter(AdapterBase):
+    def __init__(
+        self,
+        input_dir: Path,
+        output_dir: Path,
+    ):
+        super().__init__(input_dir, output_dir)
+        
+    def _convert_colmap_to_mvsnet(self):
+        base_command = "python3 colmap2mvsnet_acm.py"
```

Comment:
Why can't we just import this module and explicitly run it here without creating an additional process?

Reply:
> Why can't we just import this module and explicitly run it here without creating an additional process?

In this case, we need to edit the developers' code, since they use a parser and the code will be more cumbersome.